deploy/charts/external-secrets/values.yaml

```diff
@@ -361,14 +371,16 @@ webhook:
 
   podLabels: {}
 
-  podSecurityContext: {}
+  podSecurityContext:
+    enabled: false
```

Comment:
this would mean this helm chart is a breaking change for any users with a podSecurityContext already in place, as now the implicit opt-in is no longer seen (i.e. if people added anything to their podSecurityContext). Can the logic be less breaking?

Reply:
it can be set as `true` and just render an empty `dict`

Reply:
that would be perfect :)